ui/packages/catalog-ui-search/src/main/webapp/react-component/map-info/formatting.tsx

```diff
@@ -22,9 +22,16 @@ const usngPrecision = 6
 
 import { Attribute, Coordinates, Format, validCoordinates } from '.'
 
-export const formatAttribute = ({ name, value }: Attribute): string => {
-  const isDate = metacardDefinitions.metacardTypes[name].type === 'DATE'
-  return `${name.toUpperCase()}: ${
+export const formatAttribute = ({ name, value }: Attribute): string | null => {
```

Comment:
:pencil2: Could probably rename this now
```suggestion
export const formatAttribute = ({ attribute, value }: Attribute): string | null => {
```